apps/wing/src/cli.ts

```diff
@@ -31,7 +31,7 @@ function runSubCommand(subCommand: string) {
         process.exit(1);
       }
     } catch (err) {
-      console.error(err);
+      console.error((err as any).message);
```

Comment:
not sure we can guarantee a message property

```suggestion
      console.error((err as any)?.message ?? err);
```